src/program/types/AwaitExpression.js

```diff
@@ -0,0 +1,20 @@
+import Node from '../Node.js';
+
+const noop = () => {};
+
+export default class AwaitExpression extends Node {
+	static removeAsync(code, transforms, async, start, callback = noop) {
+		if (transforms.asyncAwait && async) {
+			code.remove(start, start + 6);
+			callback();
+		}
+	}
+
+	transpile (code, transforms) {
+		AwaitExpression.removeAsync(code, transforms, true, this.start, () => {
+			code.insertLeft(this.argument.start, 'return ');
```

Comment:
This handles the expression as though it was a statement. See for example `async () => await 1 + await 2`.